protostar/commands/test/test_results_queue.py

```diff
@@ -1,17 +1,26 @@
+from multiprocessing.managers import ValueProxy
 from typing import TYPE_CHECKING
 
-from protostar.commands.test.test_cases import TestCaseResult
+from protostar.commands.test.test_cases import TestCaseResult, PassedTestCase
 
 if TYPE_CHECKING:
     import queue
 
 
 class TestResultsQueue:
-    def __init__(self, shared_queue: "queue.Queue[TestCaseResult]") -> None:
+    def __init__(
+        self, shared_queue: "queue.Queue[TestCaseResult]", failed: ValueProxy
```

Comment:
`failed` is a bit too ambiguous name. Maybe `has_failed_or_broken_result_shared_value`? (It's a bit long, but descriptive.)

Btw: https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/shared-values/